haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A help file viewer"
+DESCRIPTION="An application for viewing help files that use a simple html like mark-up. Unlike html, .hlp is directed at help files not a rich web experiance."
```

Comment:
Looks good so far, could you cut the DESCRIPTION lines to match our guidelines (80 char limit): https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description

Reply:
Done.